doc/libpmemobj.3.md

```diff
@@ -3,7 +3,7 @@ layout: manual
 Content-Style: 'text/css'
 title: libpmemobj(3)
 header: NVM Library
-date: pmemobj API version 2.0.0
+date: pmemobj API version 2.0.1
```

Comment:
the patch version should be bumped only in the case of "backwards-compatible bug fixes."